orchestrator/Dockerfile

```diff
@@ -29,14 +29,14 @@ RUN export GNUPGHOME="$(mktemp -d)" \
         && percona-release enable-only pdps-8.0 release
 
 RUN set -ex; \
+    microdnf update -y krb5-libs; \
```

Comment:
why don't we update all packages, rather than a specific package? it'll help us in the future with other CVEs

Reply:
@egegunes  Will we have this docker image in the official docker repo?